OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianAttachments/GuardianAttachmentServiceTests.Validations.Remove.cs

```diff
@@ -19,7 +19,7 @@ public async Task ShouldThrowValidatonExceptionOnRemoveWhenGuardianIdIsInvalidAn
         {
             // given
             Guid randomAttachmentId = Guid.NewGuid();
-            Guid randomGuardianId = default;
+            Guid randomGuardianId = Guid.Empty;
```

Comment:
@KFitz114 we should rename this to `invalidGuardianId`

Reply:
@hassanhabib updated to change variable name from "random" to "invalid".